### ISSUE TITLE
SDG 11: Rename and replace image with correct color one

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#4225]
### Dependency
- #4183 

### Overview
We need to replace the Sustainable Development Goals (SDG) images in our repository so that the SDG svgs are the correct color and follow the [United Nations' SDG guidelines](https://www.un.org/sustainabledevelopment/wp-content/uploads/2019/01/SDG_Guidelines_AUG_2019_Final.pdf). 

### Details
SDG is an acronym for the [United Nations Development Programme Sustainable Development Goals](https://www.undp.org/sustainable-development-goals).

### Action Items
- [x] In the `assets/images/sdg` directory, delete `sustainable-cities.svg`
- [x] From the HackforLA.org website drive's [HackforLA.org website/SDG/SDG images](https://drive.google.com/drive/folders/1kJdguLXnIRv13IRh_4v4lrYDG7K4FbML) folder, download the following svg: `E-WEB-Goal-11.svg`
- [x] Add the svg you just downloaded into the `assets/images/sdg` directory
- [x] Rename that svg so that it becomes `sdg11.svg`
- [x] In `_includes/about-page/about-card-sustainability.html`, update the path to the SVG you just added by changing
  ```
  <img src="/assets/images/sdg/sustainable-cities.svg" alt="11 Sustainable Cities and Communities" />
  ```
  to
  ```
  <img src="/assets/images/sdg/sdg11.svg" alt="11 Sustainable Cities and Communities" />
  ```
- [x] In `_projects/lucky-parking.md`, update the path to the SVG you just added by changing
```
sdg-image-src: /assets/images/sdg/sustainable-cities.svg
```
to
```
sdg-image-src: /assets/images/sdg/sdg11.svg
```
- [x] Using Docker, check the following pages to check that the SDG image shows up on the same spot as on the live webpages
  - Note: The size of the image may look different compared to the website. This will be addressed in issue #4229. 
  - [x] About page's Sustainability section (https://www.hackforla.org/about/#sustainability)
  - [x] Citizen Engagement page: https://www.hackforla.org/citizen-engagement
  - [x] Lucky Parking: https://www.hackforla.org/projects/lucky-parking

#### Merge Team, Technical Lead, or Product Manager
- [ ] Once the pull request associated with this issue is merged, go to each of the following issues and check if all the issues listed under the `Dependency` section are closed. If the dependencies are all closed, move those issues into the `New Issue Approval` column, remove the `Dependency` label on it, and add a `ready for dev lead` label to it.
  - [x] #4187 
  - [x] #4229

### Resources/Instructions
- [assets/images/sdg](https://github.com/hackforla/website/tree/gh-pages/assets/images/sdg) directory
- [HackforLA.org website/SDG/SDG images](https://drive.google.com/drive/folders/1kJdguLXnIRv13IRh_4v4lrYDG7K4FbML)
- [_includes/about-page/about-card-sustainability.html](https://github.com/hackforla/website/blob/gh-pages/_includes/about-page/about-card-sustainability.html)
- [_projects/lucky-parking.md](https://github.com/hackforla/website/blob/gh-pages/_projects/lucky-parking.md?plain=1)
- About page's Sustainability section: https://www.hackforla.org/about/#sustainability
- Lucky Parking: https://www.hackforla.org/projects/lucky-parking
- This issue is tracked in #4153
#### United Nations' SDG Information
- [United Nations' SDG guidelines](https://www.un.org/sustainabledevelopment/wp-content/uploads/2019/01/SDG_Guidelines_AUG_2019_Final.pdf)
- [United Nations' SDG Communications materials](https://www.un.org/sustainabledevelopment/news/communications-material/)
- [United Nations Development Programme Sustainable Development Goals](https://www.undp.org/sustainable-development-goals)

**Comment by JessicaLucindaCheng** [comment]
@ExperimentsInHonesty I'm going to be writing multiple issues to replace the SDGs in our repo and will be using this issue as a template to write the other issues. I would like to get your approval on this issue before I start writing the other ones.

**Comment by ExperimentsInHonesty** [comment]
@JessicaLucindaCheng looks good to me.

**Comment by JessicaLucindaCheng** [comment]
Once this issue is in the New Issue Approval column, a dev lead needs to review the issue and determine if it is ready for a PM to prioritize it. If it isn't ready, a dev lead will need to edit this issue so that it is ready for a PM to prioritize it.

**Comment by StephenTheDev1001** [comment]
i. Availability: Mon-Sat 4-7pm
ii. ETA: Apr 7th

**Comment by StephenTheDev1001** [comment]
Update
1. Progress: Completed all tasks under "Action Items"
2. Blockers: sdg11.svg is not displaying on Firefox.
3. Availability: Mon - Sat 4-7pm
4. ETA: Apr 10th

Working on `sdg11-replace-img-4225` branch on my fork.
@drakenguyen4000 describes the problem well [HERE with images](https://github.com/hackforla/website/pull/4437#pullrequestreview-1377091297)

**Comment by StephenTheDev1001** [comment]
I was told at the meeting that my changes are correct and it will be merged soon.
